pages/common/mupdf.md

```diff
@@ -0,0 +1,32 @@
+# mupdf
+
+> MuPDF is a lightweight PDF viewer (also supports XPS and E-Book formats).
```

Comment:
Perhaps this will read better?
```suggestion
> MuPDF is a lightweight PDF viewer, with full support for PDF, XPS and E-Book formats.
```

Reply:
I am not sure it has full support. Also, I haven't really met anyone that uses it as a XPS or E-Book reader. How about:
MuPDF is a lightweight PDF, XPS, and E-book viewer?

This is from the website of MuPDF

Reply:
Sounds perfect&mdash;
```suggestion
> MuPDF is a lightweight PDF, XPS, and E-book viewer.
```